Building Quality Assistance + Coaching with Levels 1-4 (#28)
OctopusDeploy/People: Quality-Assistance.md

> ## Quality Assistance

Reviewer: Here and in [Confluence](https://octopushq.atlassian.net/wiki/spaces/IN/pages/892370955/Quality+Assistance+Team), there's a mixture of "Quality Assistance engineer", "Quality Assistance", "quality assurance", and "quality coaching". Do they all mean different things, or are there some synonyms we can eliminate?

Author: Good question. As I've talked to people in other scenarios, they talk about the "practice of quality assistance" as a model and "quality coach" as a role.

We don't have to follow that convention. Perhaps we remove everything except "quality coach" and derivatives? What do you think @Waldo000000

Reviewer 2: I think so too. `Quality coaches` provide `quality assistance` in general. 
➕  on

[Quality Assistance at Octopus](https://octopushq.atlassian.net/wiki/spaces/IN/pages/892370955/Quality+Assistance)

# Everyone

> See [Everyone](https://github.com/OctopusDeploy/People/blob/master/Everyone.md) and these specific examples below.

## I am an egoless, team-oriented person

- I actively build good relationships with individuals in our product delivery teams, including product managers, product designers, and software engineers.
- I can manage difficult conversations and tactfully challenge others, leaving them with a positive impression of myself and Octopus as a whole.
- Even when I disagree with something, I commit to the agreed course of action, work enthusiastically toward the best outcome, and learn from the experience.
- I recognize and accept that we will always be learning and improving how we work, and my approach allows me to tactfully challenge the status quo or confidently adapt to change.

## I elevate product delivery teams

> - Product development teams genuinely want my assistance with their work.

Reviewer: 1. Is a "product delivery team" the same as a "product development team"?
2. I assume that a quality coach is embedded _within_ such team(s), but the way this is phrased implies they are on the outside looking in. If my assumption is correct, consider a tweak:

```suggestion
- The people that I'm working with in product delivery teams genuinely want my assistance with their work.
```

> - I raise the level of coordination, collaboration, and clarity in the teams I'm coaching.

Reviewer: ```suggestion
- I raise the standard of coordination, collaboration, and clarity in the teams I'm coaching.
```
(to clarify that by "raise the level of" we do not necessarily mean "raise the _amount_ of")

- My written and verbal communication is clear, concise, articulate, and persuasive.
- My contributions to team discussions are timely, net-positive, informative, and helpful.

## I am an advocate for our customers and technical quality

> - I am easily recognised as an advocate for technical quality at Octopus.

Reviewer 2: I am easily recognised as an advocate for technical and product quality at Octopus

> - I am easily recognised as an advocate for things that actually matter to our customers.

Reviewer: > I am easily recognised as an advocate for things that actually matter to our customers.

I like this as a grounding principle ❤️

- I have a broad and thorough understanding of the problems our customers are trying to solve and how we do/don't solve them, helping me make good tradeoffs with the entire picture in mind.
- In addition to our product delivery teams, I am well-connected to our other teams, like support, customer advisory, trust and security, and customer success.
- I have attained mastery in many customer problem+solution areas and am regularly helping others attain mastery themselves.
- People genuinely want my help shaping new ideas, to explore new customer problems+solutions and discover potential risks.

## I am actively improving our entire product delivery team

- I have a clear picture of the ideal product delivery team and help our teams make appropriate and timely improvements each cycle.
- I help our team clearly identify negative quality trends and help make appropriate and timely improvements each cycle.
- I raise appropriate and timely concerns in #product-delivery and contribute effectively to conversations that help clarify ways we can improve.

## I am actively improving our quality coaching team

> - I have a clear picture of the ideal quality coaching team and help make appropriate and timely improvements each cycle.

Reviewer 2: Elaborating on 
`help make appropriate and timely improvements each cycle.`
- I experiment with different quality coaching activities to identify the practices that will provide the most confidence in delivery each cycle.

Author: ```suggestion
- I help the quality coaching team improve how they do daily work so we can achieve better outcomes with the teams we coach.
```

- I confidently partner with teams using an iterative development approach, from walking skeleton to polished product.
- I confidently partner with teams through building smaller improvements within a single cycle, or complex value streams spanned across multiple cycles.
- I confidently help product teams make pragmatic decisions during the fixed timelines of our build cycles, taking the bigger picture into account.
- I confidently organise, coordinate, and run quality coaching activities, like QA KickOff and QA Demo, and how to reap the most value from these activities.

# Lead Quality Coach (Level 4)

> Building on Senior Quality Coach (Level 3), I am a force multiplier, deliberately shifting my focus from individual quality coaching toward uplifting the whole product delivery team.

## I am an expert in our customers and our business and could be an effective product manager or customer advocate

- I have a broad and thorough understanding of the problems our customers are trying to solve and how we do/don't solve them, helping me make good tradeoffs with the entire picture in mind.

> - In addition to our product delivery teams, I am well-connected to our other teams, like support, customer advisory, trust and security, and customer success.

Reviewer 2: Isn't `Customer success team` now `Customer advisory team`?

Author: Yes, and `Sales` is `Customer Success`, I think.
